### PR TITLE
Fix TV focus issues for add repo input

### PR DESCRIPTION
**Pull request by Luna712**

### CODE REVIEW

File: app/src/main/res/layout/add_repo_input.xml [hunk 1]
```diff
@@ -83,11 +81,8 @@
             android:layout_height="wrap_content"
             android:hint="@string/repository_url_hint"
             android:inputType="textUri"
-            android:nextFocusLeft="@id/apply_btt"
```
Comment (fire-light42) [inline app/src/main/res/layout/add_repo_input.xml:86]: Why did you remove left and right?

Comment (Luna712) [reply app/src/main/res/layout/add_repo_input.xml:86]: Because left and right is unexpected behavior when there is nothing to the left or right I don't expect it to jump to the bottom buttons if I accidentally press left or right, which is what it does. If you want it readded I can I just thought of it as behavior you don't expect.

Comment (fire-light42) [reply app/src/main/res/layout/add_repo_input.xml:86]: CloudStream 3 uses that behavior in a lot of places to make it easier to navigate. So no, I dont think that is unexpected behavior.

https://github.com/search?q=repo%3Arecloudstream%2Fcloudstream%20nextFocusLeft%3D%22%40id%2Fapply_btt&type=code

Comment (Luna712) [reply app/src/main/res/layout/add_repo_input.xml:86]: Readded
